core/file/dicom/element.cpp

```diff
@@ -354,7 +354,23 @@ namespace MR {
       }
 
 
+      void Element::error_in_get (size_t idx) const
+      {
+        const std::string& name (tag_name());
+        DEBUG ("value not found for DICOM tag " + printf("%04X %04X ", group, element) + ( name.size() ? name.substr(2) : "unknown" ) + " (at index " + str(idx) + ")");
+      }
 
+      void Element::error_in_check_size (size_t min_size, size_t actual_size) const
+      {
+        const std::string& name (tag_name());
+        throw Exception ("not enough items in for DICOM tag " + printf("%04X %04X ", group, element) + ( name.size() ? name.substr(2) : "unknown" ) + " (expected " + str(min_size) + ", got " + str(actual_size) + ")");
+      }
+
+      void Element::report_unknown_tag_with_implicit_syntax () const
+      {
+        DEBUG (MR::printf ("attempt to read data of unknown value representation "
+              "in DICOM implicit syntax for tag (%02X %02X) - ignored", group, element));
```

Comment:
I can see it's a copy-and-paste of code from `core/file/dicom/element.h:138`; but should these not be `%04X`?

Reply:
Yes, good point. Doesn't make much difference in practice (it won't truncate to 2 characters, just zero-pad to at least 2 characters), but best to make all this consistent. I'll amend now.